<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ACME4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>781</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>起始页</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="treeWidgetProject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>601</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <column>
     <property name="text">
      <string>项目</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工程</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>插件管理</string>
     </property>
     <addaction name="actionNewPlugin"/>
     <addaction name="actionDelPlugin"/>
    </widget>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="menu_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenProject">
   <property name="text">
    <string>打开工程</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>新建工程</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>关闭工程</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>网络设置</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>语言设置</string>
   </property>
  </action>
  <action name="actionNewService">
   <property name="text">
    <string>新建服务表</string>
   </property>
  </action>
  <action name="actionNewPlugin">
   <property name="text">
    <string>新建插件</string>
   </property>
  </action>
  <action name="actionDelPlugin">
   <property name="text">
    <string>删除插件</string>
   </property>
  </action>
  <action name="actionNewInstrument">
   <property name="text">
    <string>新建仪器</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>关闭工程</string>
   </property>
  </action>
  <action name="actionDelProject">
   <property name="text">
    <string>删除工程</string>
   </property>
  </action>
  <action name="actionDelService">
   <property name="text">
    <string>删除服务</string>
   </property>
  </action>
  <action name="actionDelInstrument">
   <property name="text">
    <string>删除仪器</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
